### PR TITLE
[DUOS-2974] Refactor the AdminEditUser Component

### PR DESCRIPTION
**Pull request by PintoGideon**
### Addresses

https://broadworkbench.atlassian.net/jira/software/c/projects/DUOS/boards/123?selectedIssue=DUOS-2974

### Summary

- Remove the usage of react-hyperscript-helpers
- Convert from class component to function component
- Update the file extensions from js to jsx
----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rushtong — APPROVED** [review]
Tested locally, no regressions 👍🏽

File: src/pages/AdminEditUser.jsx [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+                    className='f-left btn-primary btn-back'
+                  >
+                   Back
+                  </button>
+                </div>
+                <button
+                  id='btn_save'
+                  onClick={OKHandler}
+                  className='f-right btn-primary common-background'
+                  disabled={!displayNameValid}
```
Comment (aarohinadkarni) [inline src/pages/AdminEditUser.jsx:320]: I'm noticing that now, the Save button is disabled after an Institution or Role change until the Name value is changed. I'm not sure if this is intentional, but this is different between what's currently in dev.

Comment (rushtong) [reply src/pages/AdminEditUser.jsx:320]: Nice catch!

Comment (PintoGideon) [reply src/pages/AdminEditUser.jsx:320]: Thank for this catch @aarohinadkarni  It's a bug. I'll send out a fix.

**Comment by rushtong** [comment]
I believe this PR will also fix a reported production bug: https://broadworkbench.atlassian.net/browse/DUOS-3046

**Review by aarohinadkarni — APPROVED** [review]
thanks for the changes! 👍

**Comment by rushtong** [comment]
It turns out that this PR did not fix the bug described in https://broadworkbench.atlassian.net/browse/DUOS-3046

**Comment by PintoGideon** [comment]
@rushtong  Will send out a PR with a fix today. Sorry about that.

**Pull request merged**